docs/user/convert/premade/gdelt.rst

```diff
@@ -0,0 +1,66 @@
+Global Database of Events, Language, and Tone (GDELT)
+=====================================================
+
+This directory provides `GDELT <http://gdeltproject.org/>`__ GeoMesa
+ingest commands and converter configuration files.
+
+GDELT Event Database periodically scans news articles and uses natural
+language processing to identify "the people, locations, organizations,
+counts, themes, sources, emotions, counts, quotes and events driving our
+global society every second of every day."
+
+GDELT data is updated each morning at 6am. The ingester has last been
+tested on May 23rd 2016's dataset.
```

Comment:
I just ran an ingest with data from Jan 11th, 2017 if you want to update this. Had this result 

> INFO  Ingested 196879 features and failed to ingest 24935 features.